src/xdocs/config_filters.xml

```diff
@@ -443,41 +451,6 @@ public class UserService {
             suppressions file was not found the Filter accepts all audit events.
           </p>
       </subsection>
-      <subsection name="Properties" id="SuppressionFilter_Properties">
-        <div class="wrapper">
```

Comment:
https://github.com/checkstyle/checkstyle/pull/8077#issuecomment-612974457

Reply:
Notes(before the commit) used to appear before properties for all the Filters except for ```SuppressionFilter``` and ```SuppressionPlainTextCommentFilter```. So should I change ordering for every filter or just these two?